src/Snakemake/rules/VCF_fix/Filter_VCF.smk

```diff
@@ -1,14 +1,22 @@
-rule compress_and_index:
+rule compress:
     input:
         vcf="{path}/{vcf_file}.vcf",
     output:
         vcf="{path}/{vcf_file,[^/]+}.vcf.gz",
-        tbi="{path}/{vcf_file,[^/]+}.vcf.gz.tbi",
     container:
         config["singularity"].get("default", "")
     shell:
-        "bgzip {input.vcf} && tabix {output.vcf}"
+        "bgzip {input.vcf}"
 
+rule index:
+    input:
+        vcf="{path}/{vcf_file}.vcf.gz",
+    output:
+        tbi="{path}/{vcf_file,[^/]+}.vcf.gz.tbi",
+    container:
+        config["singularity"].get("default", "")
+    shell:
+        "&& tabix {input.vcf}"
```

Comment:
Ta bort &&

Reply:
Index-regeln borde kanske ha en egen .smk-fil eftersom den används på flera ställen

Reply:
> Ta bort &&

Den är borta nu

Reply:
> Index-regeln borde kanske ha en egen .smk-fil eftersom den används på flera ställen

Tänkte göra det i en seperat pull-request. Lägger in en issue för det.